modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -163,20 +163,20 @@ Perform the following steps to resolve common errors:
 . Use the `gravity plan` command to identify the phase in which the upgrade paused: 
 +
 ----
-sudo ./gravity plan
+sudo gravity plan
 ----
 +
 . Resume the upgrade using the debug flag on the phase in which the error occurred: 
 +
 ----
-sudo ./gravity upgrade --phase=< insert phase > --force --debug
+sudo gravity upgrade --phase=< insert phase > --force --debug
 ----
 +
 For example,  `--phase=/gc/rtf-controller-1`.
 . Wait for the command to run again. If the command does not terminate with an error, resume the upgrade by running the following command:
 +
 ----
-sudo ./gravity upgrade --resume
+sudo gravity upgrade --resume
```

Comment:
same comment as above (which gravity binary to use in the case of upgrades)

Reply:
in this case the new version from the `installer` directory as that is the one running the upgrade